metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -374,6 +374,16 @@ def build_corp_user_mce(
             if self.config.user_attrs_map["countryCode"] in attrs
             else None
         )
+        account_type = (
+            (attrs[self.config.user_attrs_map["accountType"]][0]).decode()
+            if self.config.user_attrs_map["accountType"] in attrs
+            else None
+        )
+        employee_id = (
```

Comment:
How universal is this field in LDAP servers? Do both Microsoft Active Directory and OpenLDAP support it?

Reply:
employeeID appears to be a pretty standart, so I added a default mapping for this ldap attribute into ldap.py as user_attrs_map["employeeId"] = "employeeID". I think this should be good enough for this attribute.

Reply:
Employee-ID is there: https://learn.microsoft.com/en-us/windows/win32/adschema/a-employeeid
The existing corpUser entity has field "departmentId", so I think "employeeId" is logically matches the place.

I agree that "accountType" is different and should not be there. Instead "employeeType" should be used, it is also standard: https://www.manageengine.com/products/ad-manager/ad-user-attributes/employeetype.html.

EmployeeType usually tells if employee account is for Employee or Contractor.

Reply:
@jjoyce0510 There is another concern we have just found. It looks like fields at customProperties will not be searchable and we need to commit a search over both fields above. Is there any way around this?

Reply:
I believe you can make CustomProperties searchable: https://github.com/datahub-project/datahub/blob/master/metadata-models/src/main/pegasus/com/linkedin/common/CustomProperties.pdl#L15

Since they have the "@Searchable" annotation!

Did you reuse this model?